volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml

```diff
@@ -17,54 +17,54 @@ spec:
     listKind: VolumeGroupClassList
     plural: volumegroupclasses
     shortNames:
-      - vgclass
```

Comment:
please realign, I don't think we need to reformat the indentations every time we will touch this file

Reply:
this is how the YAML generation does it, I think your IDE changed it at first, like [here](https://github.com/IBM/ibm-block-csi-operator/pull/292/files#diff-e97b4b6c488cb189abdde0029c0a9a8f5745029f104073ac9fe6af2feb1ffbd8)